src/views/assets/Asset/AssetDetail/PermUser.vue

```diff
@@ -133,7 +133,7 @@ export default {
       UserGroupCardConfig: {
         title: this.$t('UserGroups'),
         url: `/api/v1/assets/assets/${vm.object.id}/perm-user-groups/`,
-        detailRoute: 'UserGroupDetail',
+        detailRoute: () => import('@/views/users/Group/UserGroupDetail'),
         buttonTitle: this.$t('ViewPerm'),
         buttonClickCallback(obj) {
           vm.GenericListTableDialogConfig.visible = true
```

Comment:
The provided code does not appear to contain any irregularities, potential issues, or notable optimizations. It is quite succinct and directly implements functionality based on given inputs.

If there's anything else you need assistance with regarding JavaScript (node.js), web development, or programming theory in general? I'm here to offer help!